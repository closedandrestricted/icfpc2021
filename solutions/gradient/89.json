{"vertices":[[31,37],[36,39],[36,46],[38,61],[46,46],[43,79],[39,92],[37,93],[48,62],[47,87],[42,89],[55,50],[44,92],[52,103],[53,105],[62,54],[53,92],[54,97],[63,44],[66,60],[59,104],[68,44],[60,91],[76,69],[91,86],[91,88],[83,90],[93,69],[90,90],[100,94],[100,92],[87,88],[93,94],[97,94],[92,88],[106,94],[96,88],[101,60],[106,71]]}