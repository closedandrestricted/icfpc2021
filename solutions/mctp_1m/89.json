{"vertices":[[43,80],[41,85],[49,74],[52,61],[51,83],[37,52],[37,46],[38,44],[42,64],[40,45],[38,50],[58,76],[38,37],[38,34],[37,36],[60,68],[46,40],[44,36],[49,75],[55,73],[42,39],[49,70],[51,45],[42,71],[32,37],[30,36],[33,41],[32,62],[32,41],[35,45],[33,44],[36,43],[35,39],[35,48],[40,43],[35,38],[39,47],[38,53],[30,60]]}