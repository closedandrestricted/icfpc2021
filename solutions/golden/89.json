{"vertices":[[45,36],[40,33],[54,41],[69,45],[45,44],[78,61],[91,67],[93,66],[59,43],[82,69],[87,69],[38,51],[97,72],[98,77],[98,79],[44,57],[88,72],[92,75],[30,57],[51,59],[92,76],[33,61],[81,69],[48,72],[84,91],[82,90],[75,90],[48,89],[80,92],[84,107],[86,108],[77,94],[78,94],[82,105],[77,99],[80,110],[73,101],[43,100],[54,105]]}